app/bundles/PluginBundle/Integration/AbstractIntegration.php

```diff
@@ -2662,4 +2668,18 @@ public function formatMatchedFields($fields)
 
         return $fields;
     }
+
+    public function getLeadDonotContact($leadId)
```

Comment:
Let's add a second argument to this `$channel = 'email'` and use it in line 2677 so this can be used for other channels as we expand this support.